### ISSUE TITLE
Is NAR calculated correctly?

### ISSUE THREAD

**Issue by heneryville** [gcflymoto/LendingClub#6]
Looking at the NAR calculations, I assume it's supposed to be following that [NAR formula](https://www.lendingclub.com/public/lendersPerformanceHelpPop.action) provided by Lending Club. However, that formula is totaling valued as summed up over each monthly installment. Thus for each month of the loan the principle diminishes and is returned for the user. The formula you're using in LoanData doesn't appear to have any concept of time. It instead seems to assume that all profits and losses occur in a single month. This should give very skewed results for NAR.

Does that analysis seem correct to you?

**Comment by gcflymoto** [comment]
Yes it's not correct yet. I'd like to work on getting this addressed. Thanks for the pointer, Ill take a look

**Comment by bompus** [comment]
Any update on this?
